src/azure.js

```diff
@@ -3,7 +3,7 @@ var api         = require('./v1');
 
 api.declare({
   method:     'get',
-  route:      '/azure/:account/table/:table/read-write',
+  route:      '/azure/:account/table/:table',
```

Comment:
I guess https://tools.taskcluster.net/auth/scopes/#auth:azure-table-access/ (with the sub-scope option) is the full list of things that have the scope to use this?